.depalignrc.json

```diff
@@ -20,6 +20,10 @@
     "@typescript-eslint/parser": [
       "^5.59.0",
       "^4.33.0"
+    ],
+    "node-fetch": [
```

Comment:
I think `depalign` was silently failing for a while; I ended up fixing it but the actual issue was the lack of sinon devdependency with `depcheck`